ColyberCopter/Common/TasksGroup.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file TasksGroup.h
+ * @author Antoni Wielgus
+ * @brief 
+ * @date 2021-08-09
+ * 
+ */
+
+
+#ifndef TASKSGROUP_H
```

Comment:
One new line between brief and include guard is in our convention (small but unify the code)